src/backend/xilinx/memory_axi.rs

```diff
@@ -315,9 +331,9 @@ fn bram_logic(
     module.add_stmt(v::Parallel::Assign("DONE".into(), "bram_done".into()));
 
     // bram address logic
-    let copy_address = v::Expr::new_slice("copy_addr_offset", 4, 0);
+    let copy_address = v::Expr::new_slice("copy_addr_offset", 4, 0); // does this need to use parameterization?
```

Comment:
When you have question inside a comment, write something like:
```
XXX(nathanielnrn): ...
```
Same with TODOs:
```
TODO(username): ...
```